database.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$connection = mysqli_connect("localhost", "root", "", "books_db");
+
+if (!$connection) {
```

Comment:
- Observations: The database connection lacks error handling besides printing the error and terminating the script execution.
- Recommendations: Use a try-catch block or implement an error handler that can cleanly take care of database connection errors and log them appropriately.